src/pages/Account/Login.tsx

```diff
@@ -87,9 +87,7 @@ export const Login: React.FC<LoginProps> = ({
 
   return (
     <div className="auth-container">
-      <div className="logo-container">
-        <img src="/sparkle-header.png" alt="" width="100%" />
-      </div>
+      <div className="hero-logo github-plain"></div>
```

Comment:
HTML tags without content in them should use the shorthand tag syntax:
```suggestion
      <div className="hero-logo github-plain" />
```